src/server/submit/confirmation/index.js

```diff
@@ -1,28 +1,23 @@
+import { PageController } from '../../common/controller/page-controller/page-controller.js'
+import { Page } from '../../common/model/page/page-model.js'
+
 /**
- * Sets up the routes used in the submit confirmation page.
- * These routes are registered in src/server/router.js.
+ * @import { ServerRegisterPluginObject } from '@hapi/hapi'
  */
 
-import { submitConfirmationController } from './controller.js'
+export class ConfirmationPage extends Page {
+  sectionKey = 'confirmation'
+  urlKey = 'submit-confirmation'
+  urlPath = '/submit/confirmation'
+  view = 'submit/confirmation/index'
 
-/**
- * @satisfies {ServerRegisterPluginObject<void>}
- */
-export const submitConfirmation = {
-  plugin: {
-    name: 'confirmation',
-    register(server) {
-      server.route([
-        {
-          method: 'GET',
-          path: '/submit/confirmation',
-          ...submitConfirmationController
-        }
-      ])
-    }
-  }
+  pageTitle = 'Your animal disease movement licence application'
+  pageHeading = 'Your animal disease movement licence application'
```

Comment:
Amazed sonarcloud didn't give us a little stick for this - we've been extracting these into a const just to be sure they're identical, where they ought to be.

Reply:
there are loads of instances where we have the same two values defined like this but I'll change this one at least